CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 # CHANGELOG
 
-## Next Release
+## v5.0.0 (2023-05-15)
 
 ### Breaking Changes
```

Comment:
suggestion: We don't have convention for the `CHANGED` and `REMOVED` language in changelogs, I'd caution against using it so that we can keep the changelog uniform. If you'd like to change that, maybe you could put together a quick EPEP and get it approved by the team, then we could fix all the changelogs to match so it remains consistent.

Reply:
Sounds good. Was attempting to follow [Keep a Changelog convention](https://keepachangelog.com/en/1.1.0/)